pkgs/by-name/ju/juliaup/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "juliaup";
+  version = "v1.17.13";
+
+  src = fetchFromGitHub {
+    owner = "JuliaLang";
+    repo = pname;
```

```suggestion
    repo = "juliaup";
```
Do not rely on the repo matching the package name, it won't always match (example: https://github.com/orgs/catppuccin/repositories)